secret_hitler/app.py

```diff
@@ -688,62 +717,66 @@ async def start_president_legislative(game : Game):
 
 async def cleanup(guild):
     # Removes everything created by the setup function
+    channelHandler.setChannel(None)
     category = get_category(guild)
     if category is not None:
         for c in category.channels:
             logger.debug("Deleting channel: " + c.name)
             await c.delete()
-            logger.info("Deleted channel: " + c.name)
+            logger.debug("Deleted channel: " + c.name)
         logger.debug("Deleting category: " + category.name)
         await category.delete()
-        logger.info("Deleted category: " + category.name)
+        logger.debug("Deleted category: " + category.name)
     
     for e in await guild.fetch_emojis():
         if e.user.id == client.user.id:
             logger.debug("Deleting emoji: " + e.name)
             await e.delete()
-            logger.info("Deleted emoji: " + e.name)
+            logger.debug("Deleted emoji: " + e.name)
+    
+    logger.info("Completed cleanup of emojis and channels")
 
     
 
+async def get_channel(name, category):
+    # Returns the requested channel, creating it if it doesn't already exist
+    for c in category.channels:
+        if c.name == name:
+            return c
```

Comment:
A more Python-y way of expressing this follows, although I'm not 100% sure I got the syntax right. Mostly just adding this for (self-)educational purposes.

```
    c = next((c for c in category.channels if c.name == name), None)
    return c if c is not None else:
        # channel creation code
```